machine-learning/tutorials/PhoneCallsAnomalyDetection/PhoneCallsAnomalyDetection.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Upgrade to .NET 5.0

Reply:
updated.

---
In reply to: [537993017](https://github.com/dotnet/samples/pull/4015#discussion_r537993017) [](ancestors = 537993017)